src/scenes/CreateEvent/LanguageSelector.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { observer, inject } from 'mobx-react';
+import { defineMessages } from 'react-intl';
+import { Select, MenuItem } from '@material-ui/core';
+
+import { Section } from './components';
+
+const messages = defineMessages({
+  languagSelector: {
+    id: 'create.languagSelector',
+    defaultMessage: 'Language',
+  },
+});
+
+const LanguageSelector = observer(({ store: { createEvent } }) => (
```

Comment:
was this decided by someone to have the language selector? I was thinking we just auto set the language of the event based on the currently selected UI language.